server/app/views/applicant/ApplicantUpsellTemplate.html

```diff
@@ -12,9 +12,15 @@ <h1 th:text="#{title.applicationConfirmation}"></h1>
       ></p>
       <a
         href="/"
-        class="usa-button"
+        class="usa-button margin-y-2"
         id="another-program"
         th:text="#{link.applyToAnotherProgram}"
+      ></a>
+      <a
+        th:href="${downloadHref}"
+        class="usa-button margin-y-2"
+        id="download-pdf"
```

Comment:
optional: you don't really need an ID on this button unless you're going to reference it in tests, so you could skip that part